### ISSUE TITLE
Load models from `.obj` files

### ISSUE THREAD

**Issue by michal-nawrocki** [michal-nawrocki/python-engine-3D#3]
Instead of making models by writing down coordinates of vertices, load models from `.obj` files.

**Comment by billkouts** [comment]
Do we want our own .obj reader or should we try to find an existing solution?

**Comment by michal-nawrocki** [comment]
> Do we want our own .obj reader or should we try to find an existing solution?

I would go with our own. It's a simple thing to do. From my understanding, `.obj` is the simplest format for model files as it just holds the vertices of all triangles in a line-by-line manner. 

Also, as this project is more about learning, it would be great to get to know how those model files work, how to deal with the data.

**Comment by billkouts** [comment]
I completely agree. Are you ok with me working on it ? Or is this something you would like to do?

**Comment by michal-nawrocki** [comment]
Go for it 😃 
Would love if you could get the classic teapot model to render. Here's the .obj file:
https://graphics.stanford.edu/courses/cs148-10-summer/as3/code/as3/teapot.obj
